code/datums/quirks/neutral.dm

```diff
@@ -362,3 +362,68 @@
 
 /datum/quirk/item_quirk/colorist/add_unique()
 	give_item_to_holder(/obj/item/dyespray, list(LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
+
+/datum/quirk/gamer
+	name = "Gamer"
+	desc = "You are a hardcore gamer, and you have a need to game. You love winning and hate losing. You only like gamer food."
+	icon = "gamepad"
+	value = 0
+	processing_quirk = TRUE
+	gain_text = span_notice("You feel the sudden urge to game.")
+	lose_text = span_notice("You've lost all interest in gaming.")
+	medical_record_text = "Patient has a severe video game addiction."
+	mob_trait = TRAIT_GAMER
+	/// The last time the gamer quelled the urge to game
+	var/last_game_time
+
+/datum/quirk/gamer/add()
+	// Gamer diet
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	var/datum/species/species = human_holder.dna.species
+	species.liked_food = JUNKFOOD
+	RegisterSignal(human_holder, COMSIG_SPECIES_GAIN, .proc/on_species_gain)
+	RegisterSignal(human_holder, COMSIG_MOB_WON_VIDEOGAME, .proc/won_game)
+	RegisterSignal(human_holder, COMSIG_MOB_LOST_VIDEOGAME, .proc/lost_game)
+	RegisterSignal(human_holder, COMSIG_MOB_PLAYED_VIDEOGAME, .proc/gamed)
+
+/datum/quirk/gamer/proc/on_species_gain(datum/source, datum/species/new_species, datum/species/old_species)
+	SIGNAL_HANDLER
+	new_species.liked_food = JUNKFOOD
+
+/datum/quirk/gamer/remove()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	var/datum/species/species = human_holder.dna.species
+	species.liked_food = initial(species.liked_food)
```

Comment:
This'll mess with other quirks that change species liked food (it's on `remove()` so it'll won't often be a problem, but it's worth noting).